target/linux/ramips/dts/mt7620a_bolt_bl100.dts

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "bolt,bl100", "ralink,mt7620a-soc";
+	model = "Bolt BL100";
+
+	aliases {
+		led-boot = &led_reset;
+		led-failsafe = &led_reset;
+		led-running = &led_power;
+		led-upgrade = &led_reset;
+		label-mac-device = &ethernet;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_reset: reset {
+			function = LED_FUNCTION_BOOT;
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&gpio1 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led_power: power {
+			function = LED_FUNCTION_POWER;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio1 9 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lte1 {
+			label = "blue:lte1";
+			gpios = <&gpio1 7 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lte2 {
+			label = "blue:lte2";
+			gpios = <&gpio1 6 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lte3 {
+			label = "blue:lte3";
+			gpios = <&gpio1 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lte4 {
+			label = "blue:lte4";
+			gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_lte5 {
+			label = "red:lte5";
+			gpios = <&gpio2 24 GPIO_ACTIVE_LOW>;
+		};
+
+		led_lte6 {
+			label = "red:lte6";
+			gpios = <&gpio2 25 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wps {
+			function = LED_FUNCTION_WPS;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio2 22 GPIO_ACTIVE_LOW>;
+		};
+
+		led_voip {
+			label = "blue:voip";
+			gpios = <&gpio2 23 GPIO_ACTIVE_LOW>;
+		};
+
+		led_wlan2 {
+			function = LED_FUNCTION_WLAN_2GHZ;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_wlan5 {
+			function = LED_FUNCTION_WLAN_5GHZ;
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
```

Comment:
Does this change work instead of adding the wifi trigger in target/linux/ramips/mt7620/base-files/etc/board.d/01_leds ?

```suggestion
			gpios = <&gpio1 5 GPIO_ACTIVE_LOW>;
			linux,default-trigger = "phy0tpt";
```

Reply:
Updated